api/environments/identities/traits/models.py

```diff
@@ -40,6 +40,11 @@ class Meta:
         # issues with production deployment due to multi server configuration.
         db_table = "environments_trait"
 
+    @hook(AFTER_DELETE)
```

Comment:
Does this work? Just wondering whether `self.identity` might throw an error after delete?

Reply:
No, that works fine, but the problem is that we have bulk delete as some places as well.